frontend/src/renderer/util/autoSortStandards.svelte

```diff
@@ -0,0 +1,120 @@
+<script context="module">
+    import { maxBy, each, includes, size, filter, countBy, minBy } from "lodash";
+
+    const artObjectStandards = [
+        "image",
+        "print",
+        "object",
+        "art"
+    ];
+    const targetStandards = [
+        "target",
+        "color",
+        "colors",
+        "grid"
+    ];
+    const flatFieldStandards = [
+        "flat",
+        "field",
+        "white",
+        "blank"
+    ];
+    const darkFieldStandards = [
+        "dark",
+        "field",
+        "black"
+    ];
+
+    const probabilityScoreProperties = [
+        'artObjectProbability',
+        'targetProbability',
+        'flatFieldProbability',
+        'darkFieldProbability'
+    ];
+
+    export function autoSortImages(images, externalStack) {
+
+        let includeTarget = false;
+        if(size(images) < 6) {
+            return images;
+        } else if(size(images) >= 8) {
+            includeTarget = true;
+        }
+
+        // scan each image name (and path) for possible matches
+        each(images, function (image){
+            each(probabilityScoreProperties, function (property) {
+                image[property] = 0;
+            });
+            image.imageA = false;
+            each(artObjectStandards, function (string) {
+                if(includes(image.name, string)) {
```

Comment:
includes is case sensitive (from the w3schools article i read) so it should probably be lowercased at least